build-mfem/action.yml

```diff
@@ -73,10 +81,14 @@ runs:
         MPI="NO"
         if [[ ${{ inputs.mpi }} == 'par' ]]; then
           MPI="YES"
-          echo "[INFO] Hypre symlink: hypre -> ${{ inputs.hypre-dir }}"
-          ln -s -f ${{ inputs.hypre-dir }} hypre;
-          echo "[INFO] Metis symlink: metis -> ${{ inputs.metis-dir }}"
-          ln -s -f ${{ inputs.metis-dir }} metis-4.0;
+          if [[ -d ${{ inputs.hypre-dir }} ]]; then
```

Comment:
What is the scenario in which hypre or metis does not exist?

Reply:
If we're using a package manager-provided hypre or metis, then the folder won't exist. This is mainly used for Metis 5 on Windows, which is installed via vcpkg; cmake will pick up the vcpkg-installed Metis target.

Reply:
OK, what do you think about dropping a comment for this? It confused me because that looked like a "protection" but the is no counter measure: we don’t care if the directory does not exists.
Something like "If the directory does not exist, we assume it was installed elsewhere and don’t create the link".

Reply:
```suggestion
          # If the directory does not exist, we assume it was installed elsewhere and don’t create the link
          if [[ -d ${{ inputs.hypre-dir }} ]]; then
```